File: 3st/Jaeyoung/배열 비교하기.java
class Solution {
    public int solution(int[] arr1, int[] arr2) {
        int answer = 0;
        int arr1Sum=0;
        int arr2Sum=0;
        if(arr1.length<arr2.length) answer=-1;
        else if(arr1.length>arr2.length) answer=1;
        else if(arr1.length==arr2.length){
            for(int i=0; i<arr1.length; i++){
                arr1Sum+=arr1[i];
                arr2Sum+=arr2[i];
            }
            if(arr1Sum>arr2Sum) answer=1;
            else if(arr1Sum<arr2Sum) answer=-1;
            else answer=0;
        }
        return answer;
    }
}

File: 4st/Seohyun/programmer/배열 비교하기.js
function solution(arr1, arr2) {
  if (arr1.length !== arr2.length) {
    return arr1.length > arr2.length ? 1 : -1;
  } else {
    let arr1Sum = 0;
    let arr2Sum = 0;
    for (arr of arr1) {
      arr1Sum += arr;
    }
    for (arr of arr2) {
      arr2Sum += arr;
    }

    if (arr1Sum > arr2Sum) {
      return 1;
    } else if (arr1Sum < arr2Sum) {
      return -1;
    }
  }
  return 0;
}
